src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+html {
+  font-weight: 300;
+}
```

Comment:
Set other font related styles here, other elements will inherit it (except placeholders, you have to set `inherit` for them)